source/_components/climate.markdown

```diff
@@ -75,6 +75,28 @@ automation:
         away_mode: true
 ```
 
+### {% linkable_title Service `climate.set_home_mode` %}
+
+Turn home mode on/off for climate device
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. Else targets all.
```

Comment:
Note that the whole text and also the text for other devices reads "devices". While I agree with you that your suggestion is better stylistically, without changing this phrase everywhere, it would just be inconsistent usage.